app/assets/javascripts/discourse/app/components/user-menu/notification-item.hbs

```diff
@@ -0,0 +1,28 @@
+<li class={{this.className}}>
+  <a
+    href={{this.linkHref}}
+    title={{this.linkTitle}}
+    {{on "click" this.onClick}}
+  >
+    {{d-icon this.icon}}
+    <div>
+      {{#if this.label}}
+        {{#if this.wrapLabel}}
+          <span class={{concat "notification-label " this.labelWrapperClasses}}>
+            {{this.label}}
+          </span>
+        {{else}}
+          {{this.label}}
```

Comment:
this would probably still deserve a wrapping span, even without a class, it's better to avoid this case:

```
<div>
  foo
  <span>bar</span>
</div>
```